README.md

```diff
@@ -192,3 +192,33 @@ A big thanks to all the people and friends in the Apple Developer Academy (and n
 
 Any contribution, is welcome.
 Made with ❤️ in Italy.
+
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```

Comment:
Repo does not have any "CONTRIBUTING.md" file, and I think I won't create it in short time. It should be removed.